public/app/features/dashboard/components/SaveDashboard/forms/SaveDashboardForm.tsx

```diff
@@ -105,6 +105,7 @@ export const SaveDashboardForm = ({
                     });
                     setMessage(text);
                   }}
+                  onClick={() => reportInteraction('dashboards_autogenerate_clicked', { src: 'dashboard-changes' })}
```

Comment:
Sort of inconsistent with the rest of the PR where the callback is not inlined.

Reply:
Actually, it is not needed. I forgot to remove 😓